core/src/main/java/org/apache/accumulo/core/clientImpl/RootTabletLocator.java

```diff
@@ -56,11 +56,12 @@ public <T extends Mutation> void binMutations(ClientContext context, List<T> mut
       Map<String,TabletServerMutations<T>> binnedMutations, List<T> failures) {
     TabletLocation rootTabletLocation = getRootTabletLocation(context);
     if (rootTabletLocation != null) {
-      TabletServerMutations<T> tsm = new TabletServerMutations<>(rootTabletLocation.tablet_session);
+      TabletServerMutations<T> tsm =
+          new TabletServerMutations<>(rootTabletLocation.getTserverSession());
```

Comment:
I'm not sure if this fits on one line after this suggestion or not, but this would be a good use of the `var` keyword, to avoid the redundant long class name in the variable declaration.

```suggestion
      var tsm = new TabletServerMutations<T>(rootTabletLocation.getTserverSession());
```